storm-core/test/clj/backtype/storm/multilang_test.clj

```diff
@@ -47,7 +47,7 @@
                "test"
                {TOPOLOGY-WORKERS 20 TOPOLOGY-MESSAGE-TIMEOUT-SECS 3 TOPOLOGY-DEBUG true}
                topology)
-       (Thread/sleep 11000)
+       (Thread/sleep 31000)
```

Comment:
I ran test via my machine, and found it shuts down topology as soon as topology is submitted and activated. It seems that it couldn't even emit tuples.
(Please note that we're changing strategy of topology submission from #810)